tests/unit/views/log.vue.spec.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { nextTick } from 'vue'
+import { mount } from '@vue/test-utils'
+import { createStore } from 'vuex'
+import storeOptions from '@/store/options'
+import { createVuetify } from 'vuetify'
+import sinon from 'sinon'
+import Log from '@/views/Log.vue'
+import WorkflowService from '@/services/workflow.service'
+import User from '@/model/User.model'
+import { Tokens } from '@/utils/uid'
+
+describe('Log view', () => {
+  const owner = 'svimes'
+  const workflowName = 'thud'
+  const workflowID = `~${owner}/${workflowName}`
+  const initialFile = 'koom-valley.log'
+
+  const vuetify = createVuetify()
+  let $workflowService, store
+
+  const mountFunction = (options) => mount(Log, {
+    global: {
+      plugins: [vuetify, store],
+      mocks: { $workflowService },
+    },
+    props: {
+      workflowName,
+      initialOptions: {
+        tokens: new Tokens(workflowID),
+        file: initialFile,
+      },
+    },
+    ...options
+  })
+
+  beforeEach(() => {
+    store = createStore(storeOptions)
+    store.commit(
+      'user/SET_USER',
+      new User('cylc', [], new Date(), true, 'localhost', owner)
+    )
+    $workflowService = sinon.createStubInstance(WorkflowService)
+  })
+
+  it('issues the subscription', async () => {
+    const wrapper = mountFunction()
+    expect(wrapper.vm.jobLog).toEqual(0)
+    expect(wrapper.vm.query.variables).toMatchObject({
+      id: workflowID,
+      file: initialFile,
+    })
```

Comment:
Vitest supports both Chai assertions (e.g. `expect(...).to.deep.equal`) and these which are "Jest compatible" assertions: https://vitest.dev/api/expect.html

The [`toMatchObject()`](https://vitest.dev/api/expect.html#tomatchobject) assertion is very nice if you don't care about certain fields of an object